src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,103 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("문자열 계산기 테스트")
+class StringCalculatorTest {
+
+    @MethodSource
+    @DisplayName("덧셈 테스트")
+    @ParameterizedTest(name = "{0} + {1} = {2}")
+    void add(int x, int y, int expectedResult) {
+        int result = StringCalculator.calculate(x + " + " + y);
```

Comment:
x와 y를 따로 받아서 String으로 join 한다음에 사용하신 특별한 이유가 있나요??
아래 `complexExp(String exp, int expectedResult)` 처럼
```suggestion
    @ParameterizedTest(name = "{0} = {1}")
    void add(String expressionString, int expectedResult) {
        int result = StringCalculator.calculate(expresstionString);
```
으로 사용해도 될 것 같아서요 (x와 y가 따로 사용되는 부분이 없어보여서요)

Reply:
그렇네요 왜 이렇게 했지..? 🤔🤔🤔🤔